src/serac/physics/utilities/functional/functional.hpp

```diff
@@ -357,20 +437,312 @@ class Functional<test(trial), execution_policy> : public mfem::Operator {
    * @brief Lightweight shim for mfem::Operator that produces the gradient of a @p Functional from a @p Mult
    */
   class Gradient : public mfem::Operator {
+    // this is a temporary array implementation to be replaced by axom::Array as soon as it's ready
```

Comment:
Minor: Is it worth throwing in a `FIXME` or `TODO` here for easier searchability later?